data/openldap_to_389ds/instance.inf

```diff
@@ -0,0 +1,9 @@
+[general]
+config_version = 2
+
+[slapd]
+root_password = YOUR_ADMIN_PASSWORD_HERE
```

Comment:
Wei,
I suggest you use your own password here

Reply:
I would suggest to replace this pasword (using sed or something before the migration) with the root password defined in the testapi. But it's just a minor suggestion in this case.